src/test/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1_Test.kt

```diff
@@ -245,6 +245,83 @@ internal class IonManagedWriter_1_1_Test {
         assertEquals(expected, actual)
     }
 
+    @Test
+    fun `when a system macro is shadowed, it should be written using the system e-exp syntax`() {
+        val expected = """
+            $ion_1_1
+            $ion_encoding::((macro_table (macro make_string () "make")))
+            (:make_string)
+            (:$ion::make_string (: "a" b))
```

Comment:
```suggestion
            (:$ion::make_string (:: "a" b))
```
Right? If so, hold off on fixing it, as I have an in-progress change to address this.

Reply:
Yes, that would be a correct change, but I didn't include it because I didn't think we had implemented it yet.